aws-lambda-edge-add-security-headers/main.tf

```diff
@@ -0,0 +1,33 @@
+data archive_file lambda {
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing version constraint for provider "archive" in "required_providers"